src/components/TodoItem/index.ts

```diff
@@ -0,0 +1 @@
+export * from './TodoItem';
```

Comment:
There are no major issues found in the code. The export seems to be clear and consistent. Keep up the good work! 😊